core/server/master/pom.xml

```diff
@@ -97,6 +97,11 @@
       <artifactId>alluxio-job-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-stress-shell</artifactId>
```

Comment:
@yuzhu the flow as of now is the new master RPC depend on the shell module to execute commands which end up making RPCs to the job master process?

Reply:
yes

Reply:
Yea currently this is the only way.